views/templates/front/payment.tpl

```diff
@@ -92,12 +98,16 @@
     </div>
 </div>
 <script>
+    function decodeEscapedHtml(text) {
+        return new DOMParser().parseFromString(text, 'text/html').documentElement.textContent
+    }
+
     var blockonomics_data = JSON.stringify({
         time_period: {$time_period},
-        crypto:  JSON.parse('{json_encode($crypto) nofilter}'),
+        crypto:  JSON.parse(decodeEscapedHtml('{json_encode($crypto)}')),
```

Comment:
Using DOMParser works, however it looks like we are avoiding escaping the data. We can use [the following](https://stackoverflow.com/a/68472121) in the smarty template:
`JSON.parse('{$crypto|@json_encode|escape: 'javascript'}'.replaceAll('&quot;', '"')) `